app/routes/chatbot.js

```diff
@@ -86,33 +75,20 @@ router.post('/', (req, res) => {
     mobileCommonsProfileId: req.body.profile_id,
   });
 
-  let configured = true;
-  // Check for required config variables.
-  const settings = [
-    'GAMBIT_CMD_MEMBER_SUPPORT',
-    'GAMBIT_CMD_REPORTBACK',
-    'MOBILECOMMONS_OIP_AGENTVIEW',
-    'MOBILECOMMONS_OIP_CHATBOT',
-  ];
-  settings.forEach((configVar) => {
-    if (!process.env[configVar]) {
-      const msg = `undefined process.env.${configVar}`;
-      stathat(`error: ${msg}`);
-      logger.error(msg);
-      configured = false;
-    }
-  });
-
-  if (!configured) {
-    return res.sendStatus(500);
-  }
-
-  if (!req.body.phone) {
-    stathat('error: undefined req.body.phone');
```

Comment:
TODO:  Create a separate middleware function to valid expected body parameters. Felt relatively safe to remove for now since all incoming requests to `chatbot` are from Mobile Commons mData.